ledger/participant-integration-api/src/main/scala/platform/store/dao/events/ContractsReader.scala

```diff
@@ -53,7 +57,10 @@ private[dao] sealed class ContractsReader(
 
         dispatcher.executeSql(metrics.daml.index.db.lookupContractByKeyDbMetrics) {
           implicit connection =>
-            SQL"select participant_contracts.contract_id from #$contractsTable where #$stakeholdersWhere and contract_witness in ($readers) and create_key_hash = ${key.hash} limit 1"
+            SQL"""select pc.contract_id from #$contractsTable
+                 where #$stakeholdersWhere and contract_witness in ($readers)
```

Comment:
for consistency sake, please use `pc` to scope column names

Reply:
addressed for this and a couple other queries where the table was ambiguous